stdlib/lazy.mli

```diff
@@ -16,82 +16,59 @@
 (** Deferred computations. *)
 
 type 'a t = 'a CamlinternalLazy.t
-(** A value of type ['a Lazy.t] is a deferred computation, called
-   a suspension, that has a result of type ['a].  The special expression syntax
-   [lazy (expr)] makes a suspension of the computation of [expr], without
-   computing [expr] itself yet. "Forcing" the suspension will then compute
-   [expr] and return its result. Matching a suspension with the special pattern
-   syntax [lazy(pattern)] also computes the underlying expression and tries to
-   bind it to [pattern]:
-
-  {[
-    let lazy_option_map f x =
-    match x with
-    | lazy (Some x) -> Some (Lazy.force f x)
-    | _ -> None
-  ]}
-
-   Note: If lazy patterns appear in multiple cases in a pattern-matching, lazy
-   expressions may be forced even outside of the case ultimately selected by
-   the pattern matching. In the example above, the suspension [x] is always
-   computed.
-
-   Note: [lazy_t] is the built-in type constructor used by the compiler for the
-   [lazy] keyword.  You should not use it directly.  Always use [Lazy.t]
-   instead.
-
-   Note: {!Lazy.force} (and therefore the [lazy] pattern-matching)
-   raises the {!RacyLazy} exception if forced concurrently from
-   multiple domains and the {!Undefined} exception if forced
-   concurrently from multiple systhreads or fibers within a domain.
-   If you need to share a lazy between threads, then you need to
-   implement your own synchronisation (see in particular
-   {!Lazy.try_force}).
-   Before 5.0, forcing a value concurrently from multiple systhreads
-   without synchronisation was unsafe.
-   (@since 5.0)
-
-   Note: if the program is compiled with the [-rectypes] option,
-   ill-founded recursive definitions of the form [let rec x = lazy x]
-   or [let rec x = lazy(lazy(...(lazy x)))] are accepted by the type-checker
-   and lead, when forced, to ill-formed values that trigger infinite
-   loops in the garbage collector and other parts of the run-time system.
-   Without the [-rectypes] option, such ill-founded recursive definitions
-   are rejected by the type-checker.
+(** A value of type ['a Lazy.t] is a deferred computation, called a suspension,
+    that has a result of type ['a].  The special expression syntax [lazy
+    (expr)] makes a suspension of the computation of [expr], without computing
```

Comment:
Beware of line breaks inside `[]` code blocks, they do not display as you want in the documentation.

Reply:
Good point. Let me undo it.